libqtile/utils.py

```diff
@@ -338,6 +338,7 @@ def scan_files(dirpath, *names):
     files = defaultdict(list)
 
     for name in names:
+        dirpath = os.path.expanduser(dirpath)
```

Comment:
Maybe stick it outside the loop so it's just done once?

Reply:
Sigh. This is why I shouldn't do this when I'm tired!